warehouse/balancers/src/main/java/datawave/ingest/table/balancer/ShardedTableTabletBalancer.java

```diff
@@ -96,11 +96,7 @@ protected int getMaxMigrations() {
         try {
             String maxMigrationsProp = this.configuration.getTableConfiguration(this.tableId).get(SHARDED_MAX_MIGRATIONS);
             if (maxMigrationsProp != null && !maxMigrationsProp.isEmpty()) {
-                try {
-                    maxMigrations = Integer.parseInt(maxMigrationsProp);
-                } catch (Exception e) {
-                    log.error("Unable to parse " + SHARDED_MAX_MIGRATIONS + " value (" + maxMigrationsProp + ") as an integer.  Defaulting to " + maxMigrations);
```

Comment:
It would be nice to keep the log statement, if possible.  You could check for a NumberFormatException in the catch block below, and log the message there.

Reply:
Done!  Good catch.